common/djangoapps/third_party_auth/api/urls.py

```diff
@@ -3,14 +3,13 @@
 from django.conf import settings
 from django.conf.urls import patterns, url
 
-from .views import ProviderView, UserMappingView, UserView
+from .views import UserMappingView, UserView
 
 
 PROVIDER_PATTERN = r'(?P<provider_id>[\w.+-]+)(?:\:(?P<idp_slug>[\w.+-]+))?'
 
 urlpatterns = patterns(
     '',
-    url(r'^v0/users/(?P<identifier>[^/]+)/providers/$', ProviderView.as_view(), name='third_party_auth_providers_api'),
     url(
         r'^v0/users/{username_pattern}$'.format(username_pattern=settings.USERNAME_PATTERN),
```

Comment:
Will need to support email pattern here as well.

Reply:
I think this is covered by the existing pattern.  Note that `@`, `.`, and `+` are all included:

From lms/envs/common.py

```python
USERNAME_REGEX_PARTIAL = r'[\w .@_+-]+'
USERNAME_PATTERN = r'(?P<username>{regex})'.format(regex=USERNAME_REGEX_PARTIAL)
```

Reply:
Hmm. I thought it was configurable, but it doesn't seem to be, so I guess it's OK.